Friendly-plans/app/src/androidTest/java/pg/autyzm/friendly_plans/PlanListActivityTest.java

```diff
@@ -64,7 +64,7 @@ public void whenPlanIsAddedToDBExpectProperlyDisplayedOnRecyclerView() {
     }
 
     @Test
-    public void searchForASinglePlan() {
+    public void searchForASinglePlanExpectThatPlanAtFirstPosition() {
```

Comment:
With the correct test naming convention it would be 
whenSearchForASinglePlanExpectThatPlanAtFirstPosition, whenSearchForASinglePlanUsingOnlyOneCharacterExpectThatPlanAtFirstPosition, whenSearchForEveryPlanExpectEveryPlanToAppear It's not big issue, Please update it in your next PRs :)